pom.xml

```diff
@@ -214,39 +214,13 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <artifactId>maven-dependency-plugin</artifactId>
-                    <version>2.10</version>
-                    <executions>
-                        <execution>
-                            <id>unpack-license</id>
-                            <phase>generate-sources</phase>
-                            <goals>
-                                <goal>unpack</goal>
-                            </goals>
-                            <configuration>
-                                <artifactItems>
-                                    <artifactItem>
-                                        <!-- includes common copyright headers for sources -->
-                                        <groupId>org.kaazing</groupId>
-                                        <artifactId>community.license</artifactId>
-                                        <type>jar</type>
-                                    </artifactItem>
-                                </artifactItems>
-                                <overWriteReleases>true</overWriteReleases>
-                                <overWriteSnapshots>true</overWriteSnapshots>
-                                <failOnMissingClassifierArtifact>true</failOnMissingClassifierArtifact>
-                            </configuration>
-                        </execution>
-                    </executions>
-                </plugin>
-                <plugin>
                     <!-- To update the license, do: -->
-                    <!--     mvn com.mycila:license-maven-plugin:RELEASE:format -DskipITs -DskipTests -->
+                    <!--     mvn license:format -->
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>2.3</version>
+                    <version>2.11</version>
                     <configuration>
-                        <header>${project.build.directory}/dependency/COPYRIGHT-community</header>
+                        <header>org/kaazing/community/license/COPYRIGHT</header>
```

Comment:
The standard name of the copyright file is `COPYRIGHT` and it is no longer necessary to diverge from that because packaging it into `org/kaazing/community/license` gives it the namespace isolation we need.

Yes, this is no longer copied to the local filesystem, instead it is read directly from the classpath, hence the packaging.